frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts

```diff
@@ -0,0 +1,31 @@
+import {ImageStyle} from "~/components/entry/imagelabel/admin/images/lists/image-style";
+import {Age} from "~/components/entry/imagelabel/admin/images/lists/age";
+import {Stream} from "~/components/entry/imagelabel/admin/images/lists/stream";
+import "@elements/core/inputs/text-underline";
+import "@elements/core/lists/list-two-column";
+import "@elements/core/images/ji";
+export default {
+  title: 'Admin/Images/Settings',
+}
+
+const STR_USED = "To be used only for";
```

Comment:
Does this ever change at runtime? If not - should be in an element

Reply:
Being used in an existing dynamic element and this component changes in runtime